### DISCUSSION TITLE
Metrics changed after upgrade

### DISCUSSION THREAD

**Discussion by luddite516** [NetApp/harvest#1668] [Q&A] [answered]
I recently upgraded Harvest to 22.11.  I also upgraded the full stack of software we use-- Prometheus and Thanos.  Nothing was changed in OnTAP or in Grafana.  OnTAP has not been upgrade or patched either.

Following the upgrade, I see some changes in a few of the metrics, which can only be explained by the upgrade itself.  

Here is a screenshot of node_cpu_busy before and after the code change:
![image](https://user-images.githubusercontent.com/43886002/214336561-59b06cb4-2214-45f1-8815-e2163c7efeb6.png)

I searched Discussions and Changelog for any references to node_cpu_busy, but found none.  

I double-checked the raw Prometheus metrics, and confirmed that node_cpu_busy seems to have roughly doubled in value following the code change.  So, it is likely not Grafana doing anything strange in displaying the metrics.

Any ideas where this may be coming from?  Just want to understand if there is something I have missed.

**Comment by rahulguptajss** [comment, marked as answer]
@luddite516 Yes, This metric has changed in 22.08. There is an issue for it [here](https://github.com/NetApp/harvest/issues/1152)

We have recently published harvest exported metrics [documentation](https://netapp.github.io/harvest/22.11/ontap-metrics/#node_cpu_busy). You'll find all relevant information about this counter there.

We have changed collecting this counter from `processor` object to `system:node`. Increase in value is expected as counter calculted from `system:node` provides utilization from more subsystems than the `processor` object.

**Reply by luddite516** [reply]
Ahh, thank you very much.  I searched some areas of github for "node_cpu_busy", but I did not search Issues.

Going forward, does anyone know if there is a way to search the entire repo for a given string?  Would have saved me a question in this case.

Thanks,
-Steve

**Reply by ruanruijuan** [reply]
@luddite516, I'm able to search github while under Harvest github as the screenshot below and see the issues related to "node_cpu_busy".
![image](https://user-images.githubusercontent.com/83233519/214376116-6522d504-277f-4a15-aadf-51feaf0fda82.png)

**Reply by cgrinds** [reply]
Hey @luddite516 try using the top at the very top. It will default to searching the Harvest repo if you start from any link in the Harvest project.

<img width="626" alt="image" src="https://user-images.githubusercontent.com/242252/214380699-2a676a9d-5d76-4d7e-b6f9-8ff98200ebf9.png">
